lease/leasehttp/http.go

```diff
@@ -239,7 +239,6 @@ func TimeToLiveHTTP(ctx context.Context, id lease.LeaseID, keys bool, url string
 			return nil, derr
 		}
 	case <-ctx.Done():
```

Comment:
do we need to wait on this anymore? if ctx is cancelled, the cc.Do at line 212 will return, thus trigger line 237. Right?